R/orderly.R

```diff
@@ -1,5 +1,5 @@
-##' List source reports - that is, directories within `src/` that
-##' contain a file `orderly.R`
+##' List source reports - that is, directories `src/<reportname>` that
+##' contain one of `<reportname>.R` or `orderly.R`
```

Comment:
Is this the point where we just disallow `orderly.R` entirely, and only have `src/<reportname>/<reportname>.R`?